src/sentry/static/sentry/app/actionCreators/indicator.jsx

```diff
@@ -20,6 +20,9 @@ export function addMessage(msg, type, options = {}) {
   duration = typeof duration === 'undefined' ? DEFAULT_TOAST_DURATION : duration;
 
   let action = options.append ? 'append' : 'replace';
+  // XXX: This won't actually return the indicator since we are firing an action
+  // Behavior will be different than when calling IndicatorStore.add directly, but
+  // you can just add a new message and it will by default replace active indicator
```

Comment:
Do we need the return statement below then? Should we remove it if it's misleading